src/components/task/MemberTaskAdminModal.tsx

```diff
@@ -286,7 +306,14 @@ const MemberTaskAdminModal: React.FC<
             },
```

Comment:
Based on the previous conversation, adjust the conditions and positions accordingly.

Reply:
Thanks to review, it has been adjusted and refactored.